pkg/controller/network_policy.go

```diff
@@ -202,9 +202,58 @@ func (c *Controller) handleUpdateNp(key string) error {
 		return err
 	}
 
+	// set svc addreess_set
+	svcAsNameIPv4 := strings.Replace(fmt.Sprintf("%s.%s.service.%s", np.Name, np.Namespace, "ip4"), "-", ".", -1)
+	svcAsNameIPv6 := strings.Replace(fmt.Sprintf("%s.%s.service.%s", np.Name, np.Namespace, "ip6"), "-", ".", -1)
```

Comment:
Replace `ip4` and `ip6` with `kubeovnv1.ProtocolIPv4` and `kubeovnv1.ProtocolIPv6` to keep address set name style the same.

Reply:
> Replace `ip4` and `ip6` with `kubeovnv1.ProtocolIPv4` and `kubeovnv1.ProtocolIPv6` to keep address set name style the same.

ok